Sources/Public/Internal/SignalContext.swift

```diff
@@ -4,4 +4,5 @@ protocol SignalContext: CustomStringConvertible {
     var allAttributes: Attributes { get }
     var attributes: Attributes { get set }
     func set(faultMessage: String?)
+    func setErrorType(type: String?)
```

Comment:
I'd suggest renaming this method into `set(errorType: String?)` to make it more Swifty